### PR TITLE
Add main menu and add my module

### PR DESCRIPTION
**Pull request by alejandro-aleckmcklaiyre**
created alejandro.py module and added it in the main menu under the main.py

### CODE REVIEW

File: package/alejandro.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            os.system('cls')
+            print ("------ Comment from Esparagoza ------")   
+
+        elif choice == '5':
+           os.system('cls')
+           print ("------ Comment from Gomez ------") 
+
+        elif choice == '6':
+           os.system('cls')
+           print ("------ Comment from Mosenos ------")
```
Comment (ghost) [inline package/alejandro.py:47]: I will add my own comment

**Review by ghost — CHANGES_REQUESTED** [review]

File: package/alejandro.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+        elif choice == '2':
+            os.system('cls')
+            print ("------ Goals ------") 
+            print ("1. Graduate with a diploma in Information Technology.")
+            print ("2. Obtain a non-professional driver’s license.")
+
+        elif choice == '3':
+           os.system('cls')
+           print ("------ Comment from De Leon ------")
```
Comment (deleon-jasminerobelle) [inline package/alejandro.py]: I will add my changes.

**Review by deleon-jasminerobelle — CHANGES_REQUESTED** [review]

File: package/alejandro.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+def aleck_info():
+    while True:
+        print ("\nHi! I'm Aleck Mcklaiyre R. Alejandro")
+        print("\n---Main Menu ---")
+        print("[1] Basic Info")
+        print("[2] Goals")
+        print("[3] Comment from De Leon")
+        print("[4] Comment from Esparagoza")
+        print("[5] Comment from Gomez")
```
Comment (gomez-ashleyhermione) [inline package/alejandro.py:11]: I will add my own comment

**Review by gomez-ashleyhermione — CHANGES_REQUESTED** [review]

File: package/alejandro.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+            print ("2. Obtain a non-professional driver’s license.")
+
+        elif choice == '3':
+           os.system('cls')
+           print ("------ Comment from De Leon ------")
+           print ("Great Work!") 
+
+        elif choice == '4':
+            os.system('cls')
+            print ("------ Comment from Esparagoza ------")
```
Comment (esparagoza-mikkakette) [inline package/alejandro.py]: I'll add my comment here

**Review by esparagoza-mikkakette — CHANGES_REQUESTED** [review]

**Review by esparagoza-mikkakette — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by gomez-ashleyhermione — APPROVED** [review]

**Review by deleon-jasminerobelle — APPROVED** [review]

**Pull request merged**